widget/view.html

```diff
@@ -101,51 +102,211 @@ <h3 class="modal-title col-md-9">{{viewWidgetVars.LABEL_TITLE}}</h3>
+                                            data-ng-click="setBulkImportFlags('bulkImportEnabled')"
+                                            id="indicator-bulk-import-button" aria-hidden="false"><span
+                                                class="icon icon-import"></span>&nbsp;{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_LAUNCH_BUTTON}}
+                                        </button>
+
+                                        <!-- Add New Indicator BUTTON -->
                                         <button type="button"
-                                            class="btn-link text-decoration-none font-light pull-right"
-                                            data-ng-click="" id="add-new-indicator-type" aria-hidden="false">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_ADD_INDICATOR_TYPE_BUTTON}}</button>
+                                            class="btn-link text-decoration-none font-light pull-right italic font-size-14"
+                                            data-ng-click="setAddNewIOCFlags('addNewIOCTypeEnabled')"
+                                            id="add-new-indicator-type"
+                                            aria-hidden="false">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_ADD_IOC_TYPE_LAUNCH_BUTTON}}
+                                        </button>
                                     </div>
-                                </div>
 
-                                <!-- Search Section END  -->
+                                    <!-- Bulk Import Div START -->
+                                    <div id="bulk-import-container" data-ng-show="bulkImportEnabled"
+                                        data-ng-style="{'background': isLightTheme ? '#fafaf9' : (isSteelTheme ? '#242b35' : '#262626')}">
+                                        <label class="control-label display-block manual-input-list-container"
+                                            style="padding: 10px 20px;">
+                                            <span
+                                                class="padding-top-lg font-size-16">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_BULK_IMPORT_UPLOADER_LABEL}}</span>
+                                        </label>
+                                        <div id="upload-a-file" class="margin-top-md">
+                                            <div class="file-upload bulk-import-file-upload"
+                                                data-ng-model="uploadedFile" name="uploadFileDiv"
+                                                ngf-select="uploadFiles(uploadedFile)"
+                                                data-ngf-drop="uploadFiles(uploadedFile)"
+                                                data-accept=".csv,.txt,.pdf,.xls,.xlsx,.json,.doc,.docx"
+                                                data-ngf-pattern="'.csv,.txt,.pdf,.xls,.xlsx,.xlsm,.json,.doc,.docx'"
```

Comment:
store this pattern in js in scope variable

Reply:
Done